config_setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
```

Comment:
@vontell we should make this a proper CLI tool and package it up correctly. See here, e.g.: http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html